### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by tamdn96**

### CODE REVIEW

**Comment by tamdn96** [comment]
image

**Comment by tamdn96** [comment]
ready

File: app/views/static_pages/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<% provide(:title,  t(".title")) %>
```
Comment (tranducquoc) [inline app/views/static_pages/about.html.erb]: <% provide :title,  t(".title") %>

File: app/views/static_pages/contact.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, t(".title")) %>
```
Comment (tranducquoc) [inline app/views/static_pages/contact.html.erb]: tương tự

File: app/views/static_pages/help.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t(".title")) %>
```
Comment (tranducquoc) [inline app/views/static_pages/help.html.erb]: tương tự

**Comment by tranducquoc** [comment]
commented

**Comment by tamdn96** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged
Tự merged pull nhé

**Pull request merged**